app/assets/images/payment_icons/archarge.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 38 24" width="38" height="24" role="img" aria-labeledby="pi-archarge"><title id="pi-archarge">Archarge</title><path d="M36 24H2a2 2 0 01-2-2V2a2 2 0 012-2h34a2 2 0 012 2v20c0 1.11-.89 2-2 2z" fill="#fff"/><path d="M36 1c.55 0 1 .45 1 1v20c0 .55-.45 1-1 1H2c-.55 0-1-.45-1-1V2c0-.55.45-1 1-1h34m0-1H2a2 2 0 00-2 2v20a2 2 0 002 2h34a2 2 0 002-2V2c0-1.11-.89-2-2-2z" fill="#231f20" opacity=".07"/><path d="M26.66 19.4c-.28 0-.56-.04-.83-.12-.43-.12-1.03-.44-1.77-2.21-1.26-3.03-4.49-11.27-4.8-12.03-.26-.62-.33-.72-.46-.72s-.21.12-.45.78l-4.69 12.39c-.37 1-.79 1.75-1.73 1.87-.16.02-.45.04-.6.04-.1 0-.18.04-.18.12 0 .12.12.16.39.16.97 0 2.1-.06 2.33-.06s1.12.06 1.61.06c.18 0 .31-.04.31-.16 0-.08-.06-.12-.2-.12h-.2c-.35 0-.78-.18-.78-.58.02-.46.12-.91.29-1.33l1.24-3.53c.02-.09.11-.16.2-.16h4.43c.08-.01.16.04.18.12l1.92 4.91c.12.32 0 .5-.12.54-.08.02-.14.06-.14.14 0 .12.22.12.62.14 1.42.04 2.93.04 3.24.04.23 0 .41-.04.41-.16-.02-.11-.1-.13-.22-.13zm-6.17-6.32h-3.94c-.06 0-.1-.04-.06-.14l1.81-5.38c.11-.3.2-.3.31 0l1.96 5.38c.03.08.03.14-.08.14z" fill="#1e372f"/></svg>
```

Comment:
The border radius looks different compared to other icons.

[Video demonstration](https://screenshot.click/27-56-76l7p-a3pf2.mp4)

I assume it has to do with the border radius set inside vs. outside in your illustration tool

Reply:
I am submitting this on Arhaus behalf and will ask them to send me a new one today.

Reply:
Hey @tauthomas01 I added the updated payment icon with the border radius changed. Let me know if you see the updated file or I can send directly for the sake of getting things moving, in case I messed up the update

Reply:
It works. `arhaus.svg` has the right border radius.

Can we delete `archarge.svg` since it's the old one?

Reply:
@tauthomas01 yes delete anything you do not need

Reply:
OK I will close this and try to recreate then.